src/dsp_tools/commands/validate_data/models/data_deserialised.py

```diff
@@ -64,7 +64,7 @@ class AudioSegmentDeserialised(AbstractResource): ...
 
 
 @dataclass
-class ValueDeserialised:
+class ValueDeserialised(ABC):
```

Comment:
Why is this an ABC? It has no abstract methods that must be overridden in child classes